### ISSUE TITLE
New update causes iPad snapshot automation to fail when running alongside other iPhone snapshot automations

### ISSUE THREAD

**Issue by saoudrizwan** [fastlane/fastlane#12251]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I've been using snapshot automation for weeks now without a problem, but after the most recent update, the iPad version of the app crashes while the iPhone simulators show no problems. (I opened the simulator during the tests to figure out what was happening as the iPad app simply wouldn't even open for the tests.) If I run the UI test for the iPad manually, everything works fine as expected. It only ever crashes before starting up when fastlane launches it for a test involving other simulators as well. If I run snapshot automation for just the iPad, then no problems occur.
